src/Nethermind/Nethermind.JsonRpc.Test/Modules/ParityModuleTests.cs

```diff
@@ -52,7 +53,9 @@ namespace Nethermind.JsonRpc.Test.Modules
     [TestFixture]
     public class ParityModuleTests
     {
-        private IParityModule _parityModule;
+        private IParityModule _parityModuleA;
+        private IParityModule _parityModuleB;
+        private IParityModule _parityModuleC;
```

Comment:
parity module B & C are used only in 1 test, they can be constructed there as variables